addon/components/ember-table/component.js

```diff
@@ -91,7 +91,7 @@ export default Component.extend({
     return {
       columns: null,
       registerColumnTree: this.registerColumnTree.bind(this),
-      tableId: this.elementId,
+      tableId: `${this.elementId}-overflow`,
```

Comment:
@mixonic I believe that this update should keep row and column occlusion working as before. Yes, the scrolling element has changed, but I believe that this is the reference used for what element to observe the scroll on.